README.md

```diff
@@ -1,7 +1,7 @@
 DEMO: https://zaneray.github.io/carharrt-styleguide/
 
-# carharrt-styleguide
```

Comment:
@cbaxter713 I noticed that this repo has a typo in the name... it's currently Carharrt (double R) but should be Carhartt (double T). I corrected any instances of the brand name within the project, but we should probably rename the repo after this branch is merged in.